server/tidb_test.go

```diff
@@ -2265,6 +2303,92 @@ func TestTopSQLStatementStats4(t *testing.T) {
 	}
 }
 
+func TestTopSQLResourceTag(t *testing.T) {
+	ts, _, tagChecker, _, cleanFn := setupForTestTopSQLStatementStats(t)
```

Comment:
`setupForTestTopSQLStatementStats` is only called for this test case. Could you enable the check for all requests performed by TiDB? Otherwise, I guess it is still very likely that some requests are not caught (or let's say, covered) by this check.